### DISCUSSION TITLE
Inpaint/outpaint - Image prompt

### DISCUSSION THREAD

**Discussion by riazuddinshaik** [mrhan1993/Fooocus-API#383] [General]
Hi. I tried to generate the printing name and logo using "mix image prompt and inpaint" feature. The text and logo generated as weird. It UNREADABLE. How to generate atleast 70% of graphics? Thankyou.

**Comment by riazuddinshaik** [comment]
@mrhan1993 Hi. I tried to generate the printing name and logo using "mix image prompt and inpaint" feature. The text and logo generated as weird. It UNREADABLE. How to generate atleast 70% of graphics? Thankyou.

**Comment by mrhan1993** [comment]
I don't fully understand your question. Maybe you can provide your generation parameters and an example of the expected effect.
